src/dialects/abstract/query-interface-typescript.ts

```diff
@@ -19,9 +21,67 @@ export class AbstractQueryInterfaceTypeScript {
    *
    * @param schema
    * @param options
+   *
+   * @returns
    */
   async createSchema(schema: string, options?: CreateSchemaOptions): Promise<void> {
     const sql = this.queryGenerator.createSchemaQuery(schema, options);
     await this.sequelize.queryRaw(sql, options);
   }
+
+  /**
+   * Drops the specified schema (table).
+   *
+   * @param schema The schema to query. Applies only to Postgres.
+   * @param options
+   *
+   * @returns
+   */
+  async dropSchema(schema: string, options?: QueryRawOptions): Promise<void> {
+    const dropSchemaQuery: string | QueryGeneratorDropSchemaQueryObject = this.queryGenerator.dropSchemaQuery(schema);
+
+    let sql: string;
+    let queryRawOptions = options;
+    if (typeof dropSchemaQuery === 'string') {
+      sql = dropSchemaQuery;
+    } else {
+      sql = dropSchemaQuery.query;
+
+      // QueryRawOptions doesn't take undefined bind
+      const bind = dropSchemaQuery.bind as BindOrReplacements;
+      queryRawOptions = { ...options, bind };
+    }
+
+    await this.sequelize.queryRaw(sql, queryRawOptions);
+  }
+
+  /**
+   * Drop all schemas
+   *
+   * @returns
+   */
+  async dropAllSchemas(): Promise<void> {
```

Comment:
You should pass options here as well, see the following test
https://github.com/sequelize/sequelize/blob/18da38c0f35de20a6c5601146156700ecf249243/test/integration/query-interface.test.js#L26-L40